src/pal/tests/palsuite/paltestlist.txt

```diff
@@ -245,7 +245,6 @@ c_runtime/vfprintf/test6/paltest_vfprintf_test6
 c_runtime/vfprintf/test7/paltest_vfprintf_test7
 c_runtime/vfprintf/test8/paltest_vfprintf_test8
 c_runtime/vfprintf/test9/paltest_vfprintf_test9
-c_runtime/vprintf/test1/paltest_vprintf_test1
```

Comment:
Do we have any way to disable tests from running conditionally? I didn't think we did?